versioned_sidebars/version-2.0-2.4-sidebars.json

```diff
@@ -1315,6 +1315,7 @@
                 }
               ]
     },
-        "contribute-to-rancher"
+        "contribute-to-rancher",
+    "glossary"
```

Comment:
```suggestion
        "contribute-to-rancher",
        "glossary"
```

Reply:
Thanks for reviewing Marty, updated this in https://github.com/rancher/rancher-docs/pull/1421/commits/25450ce3f083793b899f5bcf8bf1fa0e93fc17e3.